src/client/components/banner/banner.test.tsx

```diff
@@ -0,0 +1,30 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Banner from './banner';
+
+describe('Banner', () => {
+  it('should render link if env var is defined', async () => {
+    // given
+    process.env.REACT_APP_EOP_BANNER_TEXT = 'This is a banner text'
+    render(<Banner />);
+
+    // when
+    const banners = await screen.queryAllByText('This is a banner text');
```

Comment:
```suggestion
    const banners = screen.queryAllByText('This is a banner text');
```
`query...` queries are not async.

However, I would suggest to use `find...` queries (which are async) instead, where possible:
```tsx
// when
const banner = await screen.findByText('This is a banner text');

// then
expect(banner).toBeInTheDocument();
```

Also applies to the other PR.

Reply:
done